README.md

```diff
@@ -106,6 +106,8 @@ config.json.sample:
   "win" : {
     "title" : "Loopline Systems",
     "version" : "x.x.x.x",
+    "publisher": "Publisher Info",
+    "size": "Estimated Size",
```

Comment:
Can we add an example value here -> `200MB`?
The sample file shows mostly concrete values over the place. :)